frontend/build-configs/.prettierrc

```diff
@@ -0,0 +1,9 @@
+{
+  "semi": false,
+  "singleQuote": false,
+  "arrowParens": "avoid",
+  "printWidth": 120,
+  "bracketSameLine": true,
+  "tabWidth": 4,
+  "endOfLine": "auto"
+}
```

Comment:
I had to add a prettier config here (which is slightly different to the one in administration (`tabWidth`, `singleQuote`). Otherwise intellij always auto formats according to prettier's defaults which results in changes everywhere.
Ultimatively, we probably want a unified prettier config at the root of the repo.
Should I create an issue for that?

Reply:
Sounds good!